data/hai/unfettered side missions.txt

```diff
@@ -203,29 +139,55 @@ mission "Hai Legacy 1"
 					goto foolish
 			`	As you try to react, the Hai closest to you firmly grabs your arm and shouts through the noise. "Wait! If this ship wanted to kill you, <last>, you would already be dead."`
 			label wait
-			`	You can barely hear anything over the sound of the engines as the ship proceeds to land on the beach. Suddenly, all is calm. They turn to you, surprisingly calm. "This ship means you no harm. This is a mighty relic of the past, from the times our Empire used to control the stars: a Ladybug!`
+			`	You can barely hear anything over the sound of the engines as the ship proceeds to land on the beach. Suddenly, all is calm. They turn to you, the second one surprisingly calm but you can tell the first one was a bit tense, "This ship means you no harm," they say.`
 			choice
 				`	(Say nothing.)`
 					goto explain
 				`	"Why didn't you say anything before that happened?"`
-			`	"We could have, but where's the fun in that? We wanted to see how you would react.`
+			`	The second one speaks, "We could have, but where's the fun in that? We wanted to see how you would react.`
 			label explain
-			`	"We have a proposition for you: should you agree to bring supplies to our brethren, the Unfettered Hai, we will sell you this ship for the modest price of 6.5 million credits. You'll be able to recuperate the cost with ease by running more of these jobs. These supplies are needed on short notice, on <destination>, and we lack the time and personnel to bring them.`
+			`	"This ship is our... guarantee that nothing goes wrong, we are still not sure if we can trust you, after all," says the second one.`
+			`	The first one speaks again, after taking you to a nearby cave to show you another ship. "We would never sell you that junk. This ship, however, is a mighty relic of the past, intended for war from the times our Empire used to control the stars: an Anomalocaris!`
+			scene "ship/hai anomalocaris"
```

Comment:
Interested in making a scene for the ship in the cave? It feels a little lazy to have no scene for the Anomalocaris if we've still got one for the Ladybug.

Reply:
~~I asked enough of @RisingLeaf already I think~~
I cant do scenes so I depend on others - we could always add another scene here later, like we could in many other places of the game I think

Reply:
```suggestion
			scene "thumbnail/hai anomalocaris"
```
that said isnt it better to show the thumbnail?

Reply:
Yeah, I agree with you on the thumbnail. My concern is that having a scene for the Ladybug but not the Anomalocaris is unusual when the Anomalocaris is the focus of the story, not the Ladybug. If we can't get an Anomalocaris scene in, I'd suggest let's remove the Ladybug scene. Having both, neither, or just the Anomalocaris is (imo) preferred over just the Ladybug. It's quite confusing to only have a Ladybug scene if that's not the ship we're purchasing.

Reply:
but... its such a cool scene and the anomalocaris just is too bulky to look good in it + I *love* the ion effect on the ladybug

Reply:
![emerging_anomolocris](https://github.com/endless-sky/endless-sky/assets/94366726/00dd51fd-6f54-4728-adc8-668c126830ec)
mhhh turns out I was right for how good it looks
~~not quite the dramatic reveal I want~~

Reply:
I will say that I would be interested to make a cave scene, using the amolocoris in the current scene is basically unacceptable, but I wont get to it for the next three weeks.
So my proposal would be to leave the scene as is and then add a cave scene in a later PR.